bin/pt-stalk

```diff
@@ -1356,18 +1367,33 @@ collect_mysql_variables() {
    echo -e "\n$sql\n" >> $outfile
    $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
 
-   sql="select * from performance_schema.variables_by_thread order by thread_id, variable_name;"
-   echo -e "\n$sql\n" >> $outfile
-   $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
+   if ! _should_skip "thread-variables"; then
+      sql="select * from performance_schema.variables_by_thread order by thread_id, variable_name;"
+      echo -e "\n$sql\n" >> $outfile
+      $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
 
-   sql="select * from performance_schema.user_variables_by_thread order by thread_id, variable_name;"
-   echo -e "\n$sql\n" >> $outfile
-   $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
+      sql="select * from performance_schema.user_variables_by_thread order by thread_id, variable_name;"
+      echo -e "\n$sql\n" >> $outfile
+      $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
 
-   sql="select * from performance_schema.status_by_thread order by thread_id, variable_name; "
-   echo -e "\n$sql\n" >> $outfile
-   $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
+      sql="select * from performance_schema.status_by_thread order by thread_id, variable_name; "
+      echo -e "\n$sql\n" >> $outfile
+      $CMD_MYSQL $EXT_ARGV -e "$sql" >> $outfile
+   fi
+
+}
+
+_should_skip() {
+   local name=$1
+
+   for item in "${OPT_SKIP_COLLECTION[@]}"; do
+   echo $item >> /tmp/sveta
```

Comment:
let's implement another name here

Reply:
OMG, I need some automation to remove those print debug messages.